openeo_pg_parser_networkx/graph.py

```diff
@@ -153,62 +131,44 @@ def _parse_process_graph(self, process_graph: ProcessGraph, arg_name: str = None
                 return
         raise Exception("Process graph has no return node!")
 
-    def _parse_argument(
-        self,
-        arg: ProcessArgument,
-        arg_name: str,
-        access_func: Callable,
-        real_origin_node: str = None,
-    ):
+    def _parse_argument(self, arg: any, arg_name: str, access_func: Callable):
 
         if isinstance(arg, ParameterReference):
```

Comment:
If nothing is going to happen here, do we still need the check for a parameter reference?

Reply:
Ah, actually something should happen, namely it should go into the `else` condition to add the access func - have updated!